templates/package/partials/main-image.html

```diff
@@ -0,0 +1,15 @@
+{{#if mainImage}}
```

Comment:
Do package teasers need to replicate this whole thing? Why can't it just use the regular main-image.html?